packages/mobx/src/types/observableannotation.ts

```diff
@@ -38,6 +46,67 @@ function extend_(
     )
 }
 
+function decorate_20223_(
+    this: Annotation,
+    desc,
+    context: ClassAccessorDecoratorContext | ClassFieldDecoratorContext
+) {
+    if (__DEV__) {
+        assert20223DecoratorType(context, ["accessor", "field"])
+    }
+
+    const ann = this
+    const { kind, name, addInitializer } = context
+
+    // Backwards/Legacy behavior, expects makeObservable(this)
+    if (kind == "field") {
+        addInitializer(function () {
+            storeAnnotation(this, name, ann)
+        })
+        return
```

Comment:
it would be great if there was a config option to forbid this or at least warn. seems to me that it will just be confusing when you have a code base and forget the `accessor` modifier and it just silently breaks